ci/install.sh

```diff
@@ -8,10 +8,9 @@ if [[ "${CI_TARGET}" == lint ]]; then
 fi
 
 if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
-  brew install ninja
-  brew install gettext
```

Comment:
so we don't need gettext anymore?

Reply:
The image is provisioned with the latest version of gettext already.

From the latest master build (which still uses the same macOS image): https://travis-ci.org/neovim/neovim/jobs/457894739#L150

Reply:
do you plan to remove `brew install libtool` ?

Reply:
Done!